src/class-parsely.php

```diff
@@ -694,8 +695,14 @@ private function get_categories( int $post_id, string $delimiter = '/' ): array
 		if ( $last_tag ) {
 			$tags = explode( '/', $last_tag );
 		}
-		// remove uncategorized value from tags.
-		return array_diff( $tags, array( 'Uncategorized' ) );
+
+		// Remove default category name from tags if needed.
+		$default_category_name = $this->get_default_category()->name ?? null;
+		if ( null !== $default_category_name ) {
+			return array_diff( $tags, array( $default_category_name ) );
+		}
```

Comment:
Let's move the check into `get_default_category()`, and have it always return a WP_Term. That way, it should always have a `name` set as well, or at least be an empty string - in which case, doing the array_diff doesn't need the conditional.